api/src/test/scala/uk/ac/wellcome/platform/api/WorksUtil.scala

```diff
@@ -14,19 +14,19 @@ trait WorksUtil {
   val period = Period("the past")
   val agent = Agent("a person")
 
-  def convertWorkToDisplayWork(work: IdentifiedWork) = DisplayWork(
-    work.canonicalId,
-    work.work.title,
-    work.work.description,
-    work.work.lettering,
-    work.work.createdDate,
-    work.work.creators
+  def convertWorkToDisplayWork(work: Work) = DisplayWork(
```

Comment:
Possibly one to defer to a separate patch, but maybe we should just have an `apply()` method on `DisplayWork` that takes an instance of `Work` as an argument? It’s what we have on several of the other Display classes.